In [1]:
import pandas as pd
import numpy as np

In [2]:
hos_open = pd.read_excel('./data/hos_open.xlsx')

In [4]:
hos_open['도로명전체주소']

0        대전광역시 대덕구 동서대로1777번길 2-0, 대전웰니스병원층 (송촌동,대전웰니스병원)
1       경상북도 경산시 경산로 174-0, A동 양지요양병원1~3층층 (옥산동,양지요양병원...
2                           전라남도 목포시 수강로12번길 11-1 (행복동2가)
3                          전라남도 목포시 녹색로 41,  (석현동)층 (석현동)
4                경상북도 영주시 반지미로 275-17, 새희망병원층 (가흥동,새희망병원)
                              ...                        
4625                               강원도 영월군 영월읍 중앙로 10,  층
4626                         강원도 홍천군 홍천읍 홍천로 342-10, 342층
4627                     강원도 원주시 원문로 141- ,  (단계동)층 (단계동)
4628                              강원도 인제군 인제읍 비봉로 19, 19층
4629                   대전광역시 유성구 온천동로 3-0, 유성웰니스병원층 (봉명동)
Name: 도로명전체주소, Length: 4630, dtype: object

In [7]:
address = hos_open['도로명전체주소'].str.split('(').str[0]
address

0       대전광역시 대덕구 동서대로1777번길 2-0, 대전웰니스병원층 
1       경상북도 경산시 경산로 174-0, A동 양지요양병원1~3층층 
2                    전라남도 목포시 수강로12번길 11-1 
3                        전라남도 목포시 녹색로 41,  
4             경상북도 영주시 반지미로 275-17, 새희망병원층 
                       ...                 
4625                 강원도 영월군 영월읍 중앙로 10,  층
4626           강원도 홍천군 홍천읍 홍천로 342-10, 342층
4627                   강원도 원주시 원문로 141- ,  
4628                강원도 인제군 인제읍 비봉로 19, 19층
4629          대전광역시 유성구 온천동로 3-0, 유성웰니스병원층 
Name: 도로명전체주소, Length: 4630, dtype: object

In [25]:
condition = address.str.split(' ').str[1] == '전주시'
jeonju = address[condition]
jeonju

93            전라북도 전주시 덕진구 동부대로 895, 6층 
94              전라북도 전주시 덕진구 송천중앙로 125, 
95               전라북도 전주시 완산구 쑥고개로 343, 
96            전라북도 전주시 완산구 쑥고개로 398-16, 
106              전라북도 전주시 완산구 백제대로 361, 
                      ...               
1585          전라북도 전주시 완산구 노송여울2길 134-0 
1762    전라북도 전주시 완산구 세내로 285, 샘물아동병원 5층 
2255              전라북도 전주시 덕진구 떡전로 15-0 
2530       전라북도 전주시 완산구 백제대로 325, 온누리안과 
2565             전라북도 전주시 덕진구 사근1길 10-0 
Name: 도로명전체주소, Length: 124, dtype: object

- insight

1. 전주시 정보
 - 30인 이상 수용가능 병원 (상급종합병원) 개수 : 124개
 - 면적 : 206.2km²

2. 전주시 인구 정보 :  657661 명(2021.01 기준)